src/polycubectl/completion/polycubectl.sh

```diff
@@ -25,6 +25,26 @@ parse_yaml() {
 
 BASE_URL_="http://localhost:9000/polycube/v1/"
 
+# https://stackoverflow.com/a/10660730
```

Comment:
Add a comment: what is this function about?

Reply:
@mauriciovasquezbernal